packages/cli/commands/sandbox/create.js

```diff
@@ -128,7 +128,11 @@ exports.handler = async options => {
           i18n(`${i18nKey}.success.create`, {
             name,
             sandboxHubId,
-          })
+          }),
+          `
+
+The following step will prompt you to authenticate with sandbox "${name}" and add it to the CLI config.
```

Comment:
are you authenticating with a sandbox, or are you authenticating the sandbox? genuine question, not sure if there's nuance needed with the language here

Reply:
This step was removed in a later commit here so we should be all set on this